process/interface.go

```diff
@@ -232,6 +232,7 @@ type ValidatorStatisticsProcessor interface {
 	RevertPeerStateToSnapshot(snapshot int) error
 	IsInterfaceNil() bool
 	Commit() ([]byte, error)
+    RootHash() ([]byte, error)
```

Comment:
I had a comment some time ago, about us having a consistent approach for tabs vs spaces :) We see this here because the file is indented using tabs. Line 235 uses spaces (1 tab = 4 spaces). Github seems to display a bigger tab (6 spaces), so this display issue is seen only here, not in our(my) editors. I propose we have a discussion about:
1. Setting a standard
2. Every one of us setting up our IDE accordingly
3. We can set our IDE to show tabs and spaces (they show as dots and arrows) and when working on something reformat the whole code if it's not using a correct indentation

Reply:
we have some time ago decided on using tabs and changed the IDE to enforce it, but I think you were not there when we decided :)